house_rules/Roleplay_etiquette.md

```diff
@@ -0,0 +1,44 @@
+# Roleplay Etiquette
+
+## Overview
+
+The Discord is an RP (ie Roleplay) server, meaning players will interact with other players and the world as if they were their character(s). 
+The server is a safe space to expand and build upon the stories and lore of the world as well as develop player characters stories. .
+
+There is some RP etiquette Lemurian Guildmembers go by and here are some tips and tricks to understand everything that happens around the guild. 
+
+## Speaking in Character
+
+Players will note whether they are speaking in character or out of character with a combination of `"quotation marks"`, `(parathesises)`, *`italics`*, and other writting conventions. Players can format their discord messages with the formatting shown below.
+
+- `"Quotation Marks"` are used to specify when a character is speaking **in character**.
+- *`italics`* are used to describe a setting or character action **in character**
+- `(parathesises)` are used to speak **out of character**.
+- `||Spoiler Tags||` are used whenever players are trying to be stealthy. For roleplay application, you can engage with other players by rolling a stealth check and allowing others to read the obscured text if they roll a higher percpetion check.   
+- ` ```Code Blocks``` ` are used to write letters, essays, or writings **in character**. 
+- `> Indents` are sometimes used to quote someone or mark that this is something in writing.
+
+
+### Examples
+
+- `*Italics*` --> *Minimo the Mouse looks at Eligos in excitement and mounts the dashing owl.*
+- `**Bold**` --> **Eligos crashes**
+- ` ```A Letter``` ` --> ```Eligos, your medical copay is 5000gp.```
+
+
+## Tip: Communication is Key
+
+Player Characters (ie PCs) are very personal things to players. If your PC would do something that is harmful or seen as hurtful to another player, please talk to them privately first. The line between PC and player can blur sometimes and clear communication is what keeps the game fun.
+
+:::note Example 1
+> Over-protective the wizard angrily yells at Naive the rogue for leaking that Over-protective's sibling is in a devil's contract. 
+
+It can be helpful to communicate that Over-protective would be angry at Naive, not Naive's character.
+:::
```

Comment:
```suggestion
:::note Example 1

> Overious Protectivious the wizard angrily yells at Naivee the rogue for leaking that Overious's sibling is in a devil's contract. 

It can be helpful to communicate that Overious Protectivious would be angry at Naivee, not Nick who plays Naivee.
:::
```

Some name fun :) Helps with the reading a little too. Also fixed typo of player, not character.

Reply:
Overious Protectivious HAHAHAHA